docs/contributtion/experimental-code.md

```diff
@@ -0,0 +1,167 @@
+# Experimental Code
+
+The team occasionally will author code, or accept contributions, that is
+considered experimental or unstable. The goal for this code is to ship it as
+unstable for sponsor product to leverage. During this time, the team can get
+feedback around what is working and what does not work so that changes can be
+made before an official release.
+
+Experimental code is made available in the following ways:
+
+- Components and exports prefixed with `UNSTABLE_`
+- Feature flags
+- Expanding existing APIs
```

Comment:
Almost every change we make fit this category :D

Reply:
I think I should remove this. Adding a new prop to API is considered a feature. Or we should also prefix the prop with the `UNSTABLE_` prefix (or maybe `UNSAFE_` but this is a different case). But we have not met any case yet in which this approach will be useful, IMHO.